src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php

```diff
@@ -10,23 +10,41 @@
 namespace Upgrade\Infrastructure\VersionControlSystem\Generator;
 
 use Upgrade\Application\Dto\ComposerLockDiffDto;
+use Upgrade\Application\Dto\IntegratorResponseDto;
 
 class PullRequestDataGenerator
 {
     /**
      * @param \Upgrade\Application\Dto\ComposerLockDiffDto $composerDiffDto
+     * @param \Upgrade\Application\Dto\IntegratorResponseDto|null $integratorResponseDto
      * @param string|null $majorAvailableInfo
      *
      * @return string
      */
-    public function buildBody(ComposerLockDiffDto $composerDiffDto, ?string $majorAvailableInfo = null): string
-    {
+    public function buildBody(
+        ComposerLockDiffDto $composerDiffDto,
+        ?IntegratorResponseDto $integratorResponseDto,
+        ?string $majorAvailableInfo = null
+    ): string {
         $text = 'Auto created via Upgrader tool.'
             . PHP_EOL
             . PHP_EOL
             . '#### Overview'
             . PHP_EOL;
 
+        if ($majorAvailableInfo) {
+            $text .= '​' . PHP_EOL . '**Available majors:**' . PHP_EOL . PHP_EOL;
+            $text .= $majorAvailableInfo;
+        }
+
+        if ($integratorResponseDto && $integratorResponseDto->getWarnings()) {
```

Comment:
why don't we use twig instead?

Reply:
I want to add only one section here
don't want to rewrite all section